src/common/ceph_context.cc

```diff
@@ -285,11 +294,11 @@ class CephContextObs : public md_config_obs_t {
                           const std::set <std::string> &changed) override {
     if (changed.count(
 	  "enable_experimental_unrecoverable_data_corrupting_features")) {
-      ceph_spin_lock(&cct->_feature_lock);
+      ceph::spin_lock(&cct->_feature_lock);
       get_str_set(
```

Comment:
That thought's crossed my mind, too-- seemed out of scope (hah-hah!) for this PR. That said, I'm happy to do it (and things should be easy to search for now). One of the big benefits is that scoped locks are safe in the face of any thrown exceptions, and if we happen to have any case where we've missed those might "instantly" fix a bug or two.